docs/Getting-Started.md

```diff
@@ -0,0 +1,270 @@
+
+```javascript
+    var React = require('tux/React');
+    var todoActions = require('./todoActions');
+    var todoStore = require('./todoStore');
+    var Todo = require('./Todo.jsx');
+    var TodoCreateForm = require('./TodoCreateForm.jsx');
+
+    var TodoViewOwner = React.createOwnerClass({
+      getInitialState: function () {
+        return {
+          todos: todoStore.getAll() // Will get all of the Todos that are stored in the TodoStore
+        };
+      },
+
+      // This method connects this owner component to the specified store. Additionally it also adds and removes the change listeners from the store during the appropriate lifecycle events, specifically componentDidMount and componentWillUnmount.
+      connectOwnerToStore: function () {
+        return {
+          store: todoStore,
+          listener: function () {
+            this.setState({ todos: todoStore.getAll() });
+          }.bind(this)
+        };
+      },
+
+      // The registerOwnerProps method is invoked with the context of the component and allows an OwnerClass to automatically expose its properties to its child components (direct or otherwise). In this instance this component is exposing the add and remove methods specified in the return.
+      registerOwnerProps: function () {
+        // the todoActions.METHOD will take the input to that method and pass it to the dispatcher, where it will broadcast the METHOD to any store listening. If that action is registered with the store, the payload from that action will then be passed to the corresponding actions method.
+        return {
+          add: todoActions.add,
+          remove: todoActions.remove
+        };
+      },
+
+      render: function () {
+        // Create an array of todo components
+        var Todos = this.state.todos.map(function(todo) {
+          return (
+            <Todo todo={todo} />
+          );
+        });
+
+        // Render all of the Todo components and the TodoCreateForm
+        return (
+          <div>
+            <TodoCreateForm />
+            {Todos}
+          </div>
+        );
+      }
+    });
+
+    module.exports = TodoViewOwner;
+```
+
+## Todo.jsx
+`Tux` also provides the ability to build high performance components out of the box. In our Todo application within the individual Todo component, a Todo's text is a `mutableTraits` property, that means that we can check to see if this property has changed when the component will re-render. If it hasn't, we can skip the re-render since we know the component hasn't updated.
+
+A high performance component:
+
+```javascript
+    var React = require('tux/React');
+
+    var Todo = React.createMutableClass({
+      // Here we are specifying that on the props object there is key that we want to watch and see if it changes and that we only want to re-render the component when that specified prop value changes
```

Comment:
Change to: Here we are specifying that we want to watch a `text` property in our `props` for changes. We will perform a one-time deep search to find `text`.